react-frontend/src/containers/App/index.js

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
 //import Nav from '../../components/Nav';
 import NavBar from '../../components/Nav/navbar';
-import Home from '../../components/Home/home';
-import Resources from '../../components/Resources/resources';
 import Footer from '../../components/Nav/footer';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Routes from '../../components/Nav/routes';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 function App() {
```

Comment:
There's a lot of `functional` declarations. You'll find most examples in the react docs will use an alternative declaration which is a more modern way of declaring functions using the the __arrow function__

`const App = () => ()`

Reply:
Sure I can change from the function declaration to const